src/Core/src/Handlers/NavigationPage/NavigationViewHandler.Gtk.cs

```diff
@@ -13,28 +13,28 @@ public partial class NavigationViewHandler : ViewHandler<IStackNavigationView, N
 
 		protected override NavigationView CreatePlatformView()
 		{
-			return new();
+			return new NavigationView();
 		}
 
 		protected override void ConnectHandler(NavigationView nativeView)
 		{
 			base.ConnectHandler(nativeView);
-
-			var virtualView = VirtualView;
-
+			nativeView.Connect(VirtualView);
 		}
 
 		protected override void DisconnectHandler(NavigationView nativeView)
 		{
 			base.DisconnectHandler(nativeView);
-
-			var virtualView = VirtualView;
-
+			nativeView.Disconnect(VirtualView);
 		}
 
 		public static void RequestNavigation(INavigationViewHandler arg1, IStackNavigation arg2, object? arg3)
 		{
-			throw new NotImplementedException();
+			if (arg1 is NavigationViewHandler platformHandler && arg3 is NavigationRequest ea)
+			{
+				platformHandler.PlatformView?.RequestNavigation(ea);
```

Comment:
@webwarrior-ws @parhamsaremi pls rename ea to navReq or something

Reply:
Done